### ISSUE TITLE
Introduce

### ISSUE THREAD

**Issue by DivineDominion** [rougier/svg-tag-mode#32]
Somewhat related to #5, what do you think about defining a `svg-tag-face` that inherits from `:default` but allows users to customize if needed?

Am asking because with my default font setting, the tags are a bit on the larger side:

![image](https://user-images.githubusercontent.com/59080/169242732-94273a61-7c29-4302-823a-9aa212668ced.png)

Would like to make them 80% height or so. And my main font is monospaced, but for tags I'd prefer variable width.

(If there are no objections I can also try to add that myself and propose a PR)

**Comment by rougier** [comment]
You should be able to specify a different face with :face . Do you mean it is not working?

**Comment by rougier** [comment]
Like a `svg-tag-mode-default-face`?

**Comment by DivineDominion** [comment]
@rougier Yeah, something like that! (Not sure about naming conventions so I cannot suggest anything here with confidence :))

**Comment by rougier** [comment]
Done.
